src/css/components/search.scss

```diff
@@ -0,0 +1,56 @@
+@import '../core.scss';
+
+$search-btn-width:   4.5rem;
+$search-btn-height:  2rem;
+
+@mixin search-icon {
+  background-image: url('../img/i-search.svg');
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.search-bar {
+  @include clearfix;
+  max-width: none;
+  position: relative;
+  $width: $search-btn-width;
+  margin-left: 0;
+  margin-top: 0.5rem;
+  margin-right: 0;
+  margin-bottom: 0;
+
+  [type='search'],
+  .search-field {
+    -webkit-appearance: none;
```

Comment:
What does this do again? There's also https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance. Should we just do `appearance` and auto-prefixer will take care of it?